### PR TITLE
Apply black formatting

### PR DESCRIPTION
**Pull request by bryevdv**
@mattpap This one is  100% up to you, but in for a penny, in for a pound, I say. 

As I see it, doing this right now results in (roughly) only one big disruptive change in the history, instead of two distinct ones. There are 100% things I don't love about black formatting, but the whole point is that it improves diff performance and can be applied automatically and without discretion. At this stage in the project development, I'm definitely +1

FYI there are not really any knobs or dials to set. Really the only thing is line length, which is current set to 100 for both black and isort. 

So let me know if you want a different line length, or if you want to hold off on this altogether.

### CODE REVIEW

File: examples/models/file/maps.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 from bokeh.document import Document
 from bokeh.embed import file_html
-from bokeh.models import (BoxSelectTool, Circle, ColumnDataSource, GMapOptions,
-                          GMapPlot, Label, LinearAxis, MercatorTicker,
-                          MercatorTickFormatter, PanTool, WheelZoomTool,)
+from bokeh.models import (
+    BoxSelectTool, Circle, ColumnDataSource, GMapOptions, GMapPlot, Label, LinearAxis,
+    MercatorTicker, MercatorTickFormatter, PanTool, WheelZoomTool,)
```
Comment (mattpap) [inline examples/models/file/maps.py]: Trailing comma. Also there is lack of consistency between this and `glyphs.py` above.

Comment (bryevdv) [reply examples/models/file/maps.py]: ah good catch, I'll look over the examples more carefully

**Comment by mattpap** [comment]
I suppose it doesn't look too bad, though I would increase the limit to 120 characters if possible. However, the timing is pretty bad. I already have to update most/all open PRs after the sorting PR, and after this I can safely throw them all away.

**Comment by bryevdv** [comment]
@mattpap OK I will increase to 120. Just making sure: you are saying you want to wait on this until after the mypy PRs, correct?

**Comment by bryevdv** [comment]
I've bumped code to 120 and left docs/examples at 100

**Comment by bryevdv** [comment]
@mattpap what is your plan for the other open PRs? The mypy ones are pushing a month, I think if they won't be  merged soon then it will be much less work to fix them up individually than continually fix up this PR.

**Comment by mattpap** [comment]
> what is your plan for the other open PRs? 

Finish them soon, though given the number of tasks I'm working on, and their complexity, this will have to wait. Neither those PRs nor, and especially, this PR have any priority. You could have easily predicted this, discussed the schedule first and submitted such a disruptive PR at a better time.

**Comment by bryevdv** [comment]
There's never going to be a non-disruptive time for a PR like this. It just has to be gotten through. Those other PRs have been languishing for weeks. If you need help with them, you could have asked. Otherwise I'd just as soon close them at this point and come back to things later one smaller chunk at a time.

**Comment by bryevdv** [comment]
In any case I am not going to try to keep this branch up to date. The nice thing about `black` is we can just run it any time. For future reference, here is the updated `.isort.cfg`:
```
[isort]
 line_length=120
 multi_line_output=3
 include_trailing_comma=True
 use_parentheses=True
 import_heading_stdlib=Standard library imports
 import_heading_thirdparty=External imports
 import_heading_firstparty=Bokeh imports
```

and here is the new test:

```
# -----------------------------------------------------------------------------
 # Copyright (c) 2012 - 2017, Anaconda, Inc. All rights reserved.
 #
 # Powered by the Bokeh Development Team.
 #
 # The full license is in the file LICENSE.txt, distributed with this software.
 # -----------------------------------------------------------------------------

 # -----------------------------------------------------------------------------
 # Boilerplate
 # -----------------------------------------------------------------------------
 import pytest  # noqa isort:skip

 # -----------------------------------------------------------------------------
 # Imports
 # -----------------------------------------------------------------------------

 # Standard library imports
 from os import chdir
 from subprocess import PIPE, Popen

 # Bokeh imports
 from . import TOP_PATH

 # -----------------------------------------------------------------------------
 # Tests
 # -----------------------------------------------------------------------------


 @pytest.mark.codebase
 def test_black():
     """ Assures that the Python codebase passes a Black formatter check

     """
     chdir(TOP_PATH)
     proc = Popen(["black", "-l", "120", "-t", "py36", "--check", "bokeh", "tests"], stdout=PIPE, stderr=PIPE)
     _, err = proc.communicate()
     assert proc.returncode == 0, "isort issues:\n%s" % err.decode("utf-8")


 # -----------------------------------------------------------------------------
 # Support
 # -----------------------------------------------------------------------------
```

**Pull request closed without merging**